### PR TITLE
Add EventHub and EventGrid templates for Python

### PR DESCRIPTION
**Pull request by elprans**

### CODE REVIEW

File: Functions.Templates/Templates/EventGridTrigger-Python/function.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "bindings": [
+    {
+      "type": "eventGridTrigger",
+      "name": "event",
+      "direction": "in"
+    }
+  ],
+  "disabled": false
```
Comment (asavaritayal) [inline Functions.Templates/Templates/EventGridTrigger-Python/function.json]: Remove "disabled":false as the V2 runtime does not respect it anymore.

Comment (elprans) [reply Functions.Templates/Templates/EventGridTrigger-Python/function.json]: It is present in all other templates though.  Should it be removed everywhere for consistency?

Comment (asavaritayal) [reply Functions.Templates/Templates/EventGridTrigger-Python/function.json]: Yes, it needs to be removed everywhere else as well. I'll create a PR for that separately.

**Comment by elprans** [comment]
@asavaritayal Fixes pushed.

**Comment by asavaritayal** [comment]
@soninaren can we go ahead and merge this?

**Pull request merged**